eessi-2023.06-eb-4.7.2-2021b.yml

```
@@ -10,3 +10,4 @@ easyconfigs:
   - OpenMPI-4.1.1-GCC-11.2.0
   - gompi-2021b
   - FlexiBLAS-3.0.4-GCC-11.2.0.eb
+  - QuantumESPRESSO-6.8-foss-2021b.eb
```

@TopRichard It won't make a difference in terms of what gets built, but I would add a separate entry for `foss-2021b` itself, as a "checkpoint" in the easystack file:

```suggestion
  - foss-2021b.eb
  - QuantumESPRESSO-6.8-foss-2021b.eb